src/dd-win-prof/Configuration.cpp

```diff
@@ -90,6 +90,8 @@ Configuration::Configuration()
     _namedPipeName = GetEnvironmentValue(EnvironmentVariables::NamedPipeName, DefaultEmptyString);
 
     _minimumCores = GetEnvironmentValue<double>(EnvironmentVariables::CoreMinimumOverride, 1.0);
+
+    _areCallstacksSymbolized = GetEnvironmentValue<bool>(EnvironmentVariables::SymbolizeCallstacks, false);
```

Comment:
On this default value, I am still skeptical.
I think that having `true` is OK as users will have stripped symbols. Though I'm willing to be wrong & the current value is somewhat good for testing purposes.